packages/stellar-axelar-std-derive/src/storage.rs

```diff
@@ -333,9 +332,21 @@ fn contract_storage_tests(enum_name: &Ident, input: &DeriveInput) -> TokenStream
     quote! {
         #[cfg(test)]
         mod #test_module_name {
+            use super::*;
+            use prettyplease;
+            use quote::quote;
+            use syn::DeriveInput;
+
             #[test]
             fn #test_name() {
-                goldie::assert!(stringify!(#input));
+                let input = quote! {
+                    #input
+                };
+                let file: syn::File = syn::parse2(input).unwrap();
+                let formatted = prettyplease::unparse(&file)
+                    .replace("pub fn ", "\npub fn ")
+                    .replace("#[cfg(test)]", "\n#[cfg(test)]");
```

Comment:
why add these replace calls? The input is just an enum

Reply:
They're unnecessary. This was copied over from below. Replaced these with
```suggestion
                    .replace("#[instance]", "\n\t#[instance]")
                    .replace("#[persistent]", "\n\t#[persistent]")
                    .replace("#[temporary]", "\n\t#[temporary]");
```